packages/core/src/components/ui/ReviewsAndRatings/ReviewsAndRatings.tsx

```diff
@@ -1,13 +1,30 @@
 import { RatingSummary } from '@faststore/ui'
 import { usePDP } from 'src/sdk/overrides/PageProvider'
 import useScreenResize from 'src/sdk/ui/useScreenResize'
+import { type RatingSummaryProps, useUI } from '@faststore/components'
```

Comment:
```suggestion
import { type RatingSummaryProps } from '@faststore/components'
```

I think you forgot to remove since `useUI` is not being used.